ui/src/app/components/lists.tsx

```diff
@@ -3,9 +3,13 @@ import * as React from 'react';
 export const CustomBulletList = ({children}) => {
   return <ul style={{position: 'relative', listStyle: 'none', marginLeft: 0}}>{children}</ul>;
 };
-
 export const CustomBulletListItem = ({bullet, children}) => {
   return <React.Fragment>
-    <div style={{position: 'absolute', left: '-.2em'}}> {bullet} </div> <li style={{marginLeft: '1.2em'}}> {children} </li>
+    <div style={{position: 'absolute', left: '-.2em'}}> {bullet} </div>
+    <li style={{marginLeft: '1.2em'}}> {children} </li>
   </React.Fragment>;
 };
+
+export const OutsideUnorderedList = ({style = {}, children}) => {
```

Comment:
I'm not totally happy with this name because `list-style-position` is kind of an obscure property for `OutsideUnorderedList` to have a meaning but alternatives would be terribly wordy like `UnorderedListWithBulletsLeftAlignedInsteadOfWords` or something.

Reply:
Yeah, I couldn't come up with a better name offhand, so ended up going with this. I don't really like this either. Maybe `BulletAlignedUnorderedList`?

Reply:
I like `BulletAlignedUnorderedList` better.